docs/development/getting_started.rst

```diff
@@ -17,7 +17,7 @@ your development workstation.
 Ubuntu/Debian
 ~~~~~~~~~~~~~
 
-.. note:: Tested on: Ubuntu 14.04
+.. note:: Tested on: Ubuntu 14.04 and Debian sid (May 6th)
```

Comment:
It's valuable to know that it worked on that system at a given point. Sorry if the message came off rude, it wasn't intended to be. I literally don't know if there is a more specific way to refer to versions of Debian Sid than the date. If there is, I think that would be preferable here.